salt/states/junos.py

```diff
@@ -281,7 +288,7 @@ def install_config(name, **kwargs):
 
     Parameters:
       Required
-        * path or template_path (Either one of the keyword must be there):
+        * name:
```

Comment:
https://github.com/Juniper/salt/pull/26/files#diff-442286d51776ddefc791bc7f15e42fc2R272
&
https://github.com/Juniper/salt/pull/26/files#diff-442286d51776ddefc791bc7f15e42fc2R282

uses path and template_path, then we have removed this description and replaced it with name